Sources/StreamChat/Database/DTOs/MemberModelDTO_Tests.swift

```diff
@@ -72,6 +73,8 @@ class MemberModelDTO_Tests: XCTestCase {
             Assert.willBeEqual(payload.user.updatedAt, loadedMember?.userUpdatedAt)
             Assert.willBeEqual(payload.user.lastActiveAt, loadedMember?.lastActiveAt)
             Assert.willBeEqual(payload.user.extraData, loadedMember?.extraData)
+            Assert.willBeEqual(Set(payload.user.teams), loadedMember?.teams)
+            Assert.willBeFalse(payload.user.teams.isEmpty)
```

Comment:
not sure we need to verify that test data is valid since we create it manually.

Reply:
Sorry what do you mean? 🤔

Reply:
That's all about line #77 😄 Basically, the same thing mentioned in here https://github.com/GetStream/stream-chat-swift/pull/952#discussion_r607024331

Reply:
✅